engine/table/src/main/java/io/deephaven/engine/table/impl/PartitionAwareSourceTable.java

```diff
@@ -202,25 +202,22 @@ protected final BaseTable redefine(@NotNull final TableDefinition newDefinition)
                 description + "-retainColumns",
                 componentFactory, locationProvider, updateSourceRegistrar, partitioningColumnDefinitions,
                 partitioningColumnFilters);
-        final DeferredViewTable deferredViewTable = new DeferredViewTable(newDefinition, description + "-retainColumns",
+        Assert.equals(redefined.isRefreshing(), "redefined.isRefreshing()", isRefreshing(), "isRefreshing()");
```

Comment:
Why did/do we need this assertion?

Reply:
We don't need it, but technically speaking we were setting refreshing from this and not from the redefined table. We pass the redefined table in as the source table, not this; and that adds a layer of indirection. I was just being carefully lazy.

The change passes nightlies, so I'm happy to drop the assertion if you would rather it be gone.